tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/pcluster.config.update_drain.yaml

```diff
@@ -54,12 +54,13 @@ SharedStorage:
       Raid:
         Type: 0
         NumberOfVolumes: 5
+      DeletionPolicy: {{ new_raid_deletion_policy }}
   - MountDir: {{ new_ebs_mount_dir }}
     Name: /manage-ebs
     StorageType: Ebs
     EbsSettings:
       VolumeType: gp3
-      DeletionPolicy: Delete
+      DeletionPolicy: {{ new_ebs_deletion_policy }}
```

Comment:
That's a good question. The reason is that having different variables makes the test more flexible because in this way it would be ready to use different deletion policies for different storage types.

Reply:
Got it. Thanks